devops/helms/iam-org-creator/templates/configmap.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "iam-org-creator.fullname" . }}-configmap
+  labels:
+    {{- include "iam-org-creator.labels" . | nindent 4 }}
+data:
+  NESTJS_PORT: {{ .Values.iamOrgCreator.config.NESTJS_PORT | default "9000" | quote }}
+  NATS_CLIENTS_URL: {{ .Values.iamOrgCreator.config.NATS_CLIENTS_URL | default "nats://identityevents-dev-nats.energyweb.org:4222" | quote }}
+  REQUEST_NEW_ORG_ROLE: {{ .Values.iamOrgCreator.config.REQUEST_NEW_ORG_ROLE | default "mytest.roles.testy.iam.ewc" | quote }}
```

Comment:
the default for this should be`orgowner.roles.orgcreator.apps.energyweb.iam.ewc`.